### DISCUSSION TITLE
Unable to build user C modules

### DISCUSSION THREAD

**Discussion by utiq** [micropython/micropython#13296] [General]
I created a Docker file to be able to build a custom C module. I followed [this example](https://docs.micropython.org/en/latest/develop/cmodules.html#basic-example). When I build it, I never get the message 
`Including User C Module from /modules/cexample` also when I `import cexample` in python I get a module not found error. I'm building for unix

I think nothing is wrong with the build process I have created, please try it yourself, to run it type `docker-compose up --build`
[test-c-modules.zip](https://github.com/micropython/micropython/files/13791231/test-c-modules.zip)

**Comment by andrewleech** [comment]
For the make based ports it should be the folder specified, rather than the cmake for within, eg. `USER_C_MODULES=/modules`

Pointing to the cmake file is just for cmake based ports, eg. rp2 and esp.

**Comment by utiq** [comment]
🤦🏻‍♂️ it took me 2 hours to realize, thanks @andrewleech, it worked
